src/hal/src/command/raw.rs

```diff
@@ -81,11 +81,33 @@ pub enum Level {
     Secondary,
 }
 
+#[allow(missing_docs)]
+#[derive(Debug)]
+pub struct CommandBufferInheritanceInfo<'a, B: Backend> {
+    pub subpass: Option<pass::Subpass<'a, B>>,
+    pub framebuffer: Option<&'a B::Framebuffer>,
+    pub occlusion_query_enable: bool,
+    pub occlusion_query_flags: QueryControl,
+    pub pipeline_statistics: PipelineStatistic,
+}
+
+impl<'a, B: Backend> Default for CommandBufferInheritanceInfo<'a, B> {
```

Comment:
we can get rid of this block if we put `#[derive(Default)]` on top of it (as well as `QueryControl` and `PipelineStatistic`).